### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by Mrsevic** [AcuGIS/OpenTileServer#30]
I am getting numerous chown: invalid user: 'tile:tile'. I was able to fix some of similar issues but now i can't see on what lines does it stop.

**Comment by DavidGhedini** [comment]
What is the OS, which script are you using, and at what point are you getting the error?

**Comment by Mrsevic** [comment]
I am trying to run the following command:
sudo ./opentileserver.sh web carto http://download.geofabrik.de/europe/montenegro-latest.osm.pbf

I am getting the error after:

6-3. Indexing 'icesheet_outlines.shp'...
max tree depth:8
split ratio:0.55
processing data/antarctica-icesheet-outlines-3857/icesheet_outlines.shp
9994
length=35555510
version=1000
type=3
extent:box2d(-20037508.0393855161964893,-20037508.3427892550826073,20037507.2295943312346935,-8479237.0339447762817144)
 number shapes=74766
 number nodes=1891
done!

...script completed.


chown: invalid user: 'tile:tile'

I have tried with sudo, root and nothing helps. I am not sure whether or not at the point of already executing scripts i should try changing tile to tileserver. I will try to reinstall everything later on.

EDIT: I've tried and it is working but this time I got certificate error which I fixed by adding certain line but yet again I am getting stuck at:

**3-1. Downloading 'ne_110m_admin_0_boundary_lines_land'...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1589  100  1589    0     0     13      0  0:02:02  0:02:00  0:00:02   396


   Error: download failed. (error code: 500, error reason: Internal Server Error                                                                                                                                                             )**


I had to change uri's from git to https:// and to some parts i had to add sudo in front of them.

My OS version is ubuntu server ubuntu-14.04.5 with latest updates installed. User name is tileserver and ubuntu host name is ubuntu14.
